src/app.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import FlashcardForm from './flashcard-form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      flashcards: []
+    }
+    this.saveFlashcard = this.saveFlashcard.bind(this)
+  }
+
+  saveFlashcard(newCard) {
+    const flashcards = this.state.flashcards.map((card) =>
```

Comment:
You don't need to copy every object in the array, just the array itself so you can add an object.